test/parallel/test-process-resource-usage.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+const usage = process.resourceUsage();
+
+assert(usage.userCpuTimeUsedSec >= 0);
```

Comment:
I think that's a general problem. I don't know if we can reliably assume certain (supported) properties are strictly greater than zero.